packages/react-scripts/scripts/eject.js

```diff
@@ -220,8 +220,21 @@ inquirer
     }
 
     if (fs.existsSync(paths.yarnLockFile)) {
```

Comment:
If somebody has a lockfile they're likely using Yarn for development. Which means the versions they currently get might be more strict than allowed per semver. Running `npm install` on top risks giving them more fresh versions.